src/lmrtfy.js

```diff
@@ -45,4 +45,14 @@ class LMRTFY {
 }
 
 Hooks.on('ready', LMRTFY.ready)
-Hooks.on('getSceneControlButtons', LMRTFY.getSceneControlButtons)
\ No newline at end of file
+Hooks.on('getSceneControlButtons', LMRTFY.getSceneControlButtons)
+Hooks.once("ready", () => {
+    game.settings.register("lmrtfy", "useShiftedRolls", {
+        name: "SHIFTed Rolls",
+        hint: "The default for LMRTFY is to attempt the roll from the user as if they were holding [SHIFT] which prompts for mods, advantage, disadvantage. Deselecting this will send a Normal roll through directly, which may keep consistency in some systems.",
+        scope: "world",
+        config: true,
+        default: true,
+        type: Boolean
+      });
+})
```

Comment:
I wouldn't make this into a configuration option, but rather as a choice in the modifiers dropdown. The "Shifted" roll as you call it is for when the DM requests a normal roll, if they request one with advantage, it's Ctrl, with disadvantage, it's Alt. In the modifier dropdown, I'd add a "Query user" option instead and if that's chosen, then have it not use the shift/alt/ctrl modifiers when making the roll.

Reply:
Thanks, the reason I added is that it appears the 'shift' seems to be different in PF2E vs  5E. For example, this:

 ```
           fakeEvent.shiftKey = true
            fakeEvent.altKey = false;
            fakeEvent.ctrlKey = false;
```

When used in the roller.js in 5E will simply cause the dice to roll at click. When used in Pathfinder 2E it causes the Advantage/Disadvantage dialog to be shown. 

I think 'Query User' will be good, but the faked key event will have to be dynamic. That should be easy.

Reply:
Made a change to set defaultQuery at the global level to account for this, since PF2e is backwards.